glutin/src/context.rs

```diff
@@ -49,9 +49,9 @@ pub trait NotCurrentGlContext: Sealed {
     /// guaranteed to be current.
     fn treat_as_possibly_current(self) -> Self::PossiblyCurrentContext;
 
-    /// Make [`Self::Surface`] on the calling thread producing the
-    /// [`Self::PossiblyCurrentContext`] indicating that the context could
-    /// be current on the thread.
+    /// Make context current on the calling thread producing the
+    /// [`Self::PossiblyCurrentContext`]. The `surface` is used as a target for
+    /// default framebuffer.
```

Comment:
Maybe:
```suggestion
    /// Make context current on the calling thread and produce the
    /// [`Self::PossiblyCurrentContext`]. The `surface` is used as the
    /// default framebuffer.
```

(Otherwise `as a target for **the** default framebuffer`)

Reply:
maybe just drop the `producing` part at this point? I'd probably use the `and change it to [Self...`.

surface is not a default framebuffer, it's a target where framebuffer goes, you can change surfaces and have the same buffer iirc.

Reply:
Yeah, let's drop `producing` and use `change it to` or just `and return PossiblyCurrentContext`.

---

Surface isn't a framebuffer, but the word "target" isn't mentioned anywhere in https://registry.khronos.org/EGL/sdk/docs/man/html/eglMakeCurrent.xhtml nor https://www.khronos.org/opengl/wiki/Default_Framebuffer, so I'd refrain from using it.

The word `target` is often used as parameter in the binding slot for various `glBind*(GLenum target, ...)` calls however.